pyworkflow/gui/project/viewdata.py

```diff
@@ -439,7 +434,7 @@ def _openProtocolFormFromId(self, protId):
     def _openProtocolForm(self, prot):
         """Open the Protocol GUI Form given a Protocol instance"""
 
-        w = gui.form.FormWindow(Message.TITLE_NAME_RUN + prot.getClassName(), prot,
+        w = gui.form.FormWindow(pwutils.Message.TITLE_NAME_RUN + pwprot.getClassName(), prot,
```

Comment:
shouldn't pwprot.getClassName() be prot.getClassName()

Reply:
Yes, a mistake caused by find/replace :(